docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -153,6 +153,15 @@ quarkus.http.cors.exposed-headers=Content-Disposition
 quarkus.http.cors.access-control-max-age=24H
 ----
 
+[TIP]
+====
+When CORS functionality is activated your filters - e.g. ContainerRequestFilter or ContainerResponseFilter - will not be invoked.
```

Comment:
I'm not sure it's precise enough.

I suppose they are not enabled for the CORS requests but they are still enabled for standard requests? If not, I don't think this is expected.

Reply:
What I experienced with quarkus is. 
When there are no CORS your filters will be invoked for every request.

When you got CORS your filters are not used for CORS request. That is what I am trying to explain here.

Reply:
ps. I am just trying first to detail what I am trying to cover to hear from you. I will improve the text later! :D